lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -787,21 +787,12 @@ def load_image(self):
         '''
         try:
             self.log("Opening image %s" % self.load_path)
-            image_tar = open(self.load_path, 'rb')
-        except Exception as exc:
-            self.fail("Error opening image %s - %s" % (self.load_path, str(exc)))
-
-        try:
-            self.log("Loading image from %s" % self.load_path)
-            self.client.load_image(image_tar)
+            with open(self.load_path, 'rb') as image_tar:
+                self.log("Loading image from %s" % self.load_path)
+                self.client.load_image(image_tar)
         except Exception as exc:
```

Comment:
To restore almost the old behavior, add `import errno` at the top and apply this change:
```suggestion
        except EnvironmentError as exc:
            if exc.errno == errno.ENOENT:
                self.fail("Error opening image %s - %s" % (self.load_path, str(exc)))
            self.fail("Error loading image %s - %s" % (self.name, str(exc)))
        except Exception as exc:
```